doc/Programming-guide/Key-platform-features/Events/index.md

```diff
@@ -4,4 +4,854 @@ title: Events
+
+Admin REST API provides the following actions:
+
+* [Get event class families]({{root_url}}Programming-guide/Server-REST-APIs/#get-event-class-families/#TODO)
+* [Get event class family]({{root_url}}Programming-guide/Server-REST-APIs/#get-event-class-family/#TODO)
+* [Create/Edit event class family]({{root_url}}Programming-guide/Server-REST-APIs/#createedit-event-class-family/#TODO)
+* [Add event class family schema]({{root_url}}Programming-guide/Server-REST-APIs/#add-event-class-family-schema/#TODO)
+* [Get event classes]({{root_url}}Programming-guide/Server-REST-APIs/#get-event-classes/#TODO)
+* [Get application event family maps]({{root_url}}Programming-guide/Server-REST-APIs/#get-application-event-family-maps/#TODO)
+* [Get application event family maps by application token]({{root_url}}Programming-guide/Server-REST-APIs/#get-application-event-family-maps-by-application-token/#TODO)
+* [Get application event family map]({{root_url}}Programming-guide/Server-REST-APIs/#get-application-event-family-map/#TODO)
+* [Create/Edit application event family map]({{root_url}}Programming-guide/Server-REST-APIs/#createedit-application-event-family-map/#TODO)
+* [Get vacant event class families]({{root_url}}Programming-guide/Server-REST-APIs/#get-vacant-event-class-families/#TODO)
+* [Get vacant event class families by application token]({{root_url}}Programming-guide/Server-REST-APIs/#get-vacant-event-class-families-by-application-token/#TODO)
+* [Get application event class families]({{root_url}}Programming-guide/Server-REST-APIs/#get-application-event-class-families/#TODO)
+* [Get application event class families by application token]({{root_url}}Programming-guide/Server-REST-APIs/#get-application-event-class-families-by-application-token/#TODO)
+
+
+## Kaa Events Admin UI
+
+#### Managing event class families
+
+> **NOTE:** this functionality available for Tenant admin
+
+To use the Kaa events feature for one or more applications, the tenant admin should create an event class family (ECF). Each ECF should be described using the Avro format. 
+
+To create a new ECF, do the following:
+
+1. Open the **Event class families** window by clicking the corresponding link on the navigation panel.
+2. In the **Event class families** window, click **Add ECF**. 
+![](images/admin_ui/event_class_family/ecf1.png)
+3. In the **Add ECF** window, fill in all the required fields and then click **Add**.  
+**NOTE:** _the namespace and class name values should be unique._
+![](images/admin_ui/event_class_family/ecf2.png)
+4. In the **Event class family** window, add (optionally) an ECF schema by clicking **Add schema** under the **Schemas** table. 
+![](images/admin_ui/event_class_family/ecf3.png)
+5. In the **Add event class family schema** window, create an ECF schema either by using the **Event class family schema** schema form or by uploading the schema from a file, then click **Add**.
```

Comment:
Should **Add event class family schema** window be modified in Admin UI, as it has empty fields: _Version_, _Created by_ and _Date/Time created_?